From 00f6fb5d07f21c133f2351d42b63c1ab4e5a8893 Mon Sep 17 00:00:00 2001
From: Fugang Duan <fugang.duan@nxp.com>
Date: Thu, 15 Jun 2017 17:33:46 +0800
Subject: [PATCH 1899/5242] MLK-15096 arm64: dts: imx8qm/imx8qxp: enable enet2
 port

commit  ce305e3748539c95ae714a5dc1544e37ea9e39a6 from
https://source.codeaurora.org/external/imx/linux-imx.git

Since the enet clock issue is fixed at SCU code by commit 07dcb2d71dd1
("Fix bug clock enable code. Ensure that the appropriate parent PLL is
enabled/disabled.") in scu mainline branch, then enable enet2 port for
imx8qm and imx8qxp lpddr4 arm2 boards.

Signed-off-by: Fugang Duan <fugang.duan@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../boot/dts/freescale/fsl-imx8qm-lpddr4-arm2.dts  |    2 +-
 .../boot/dts/freescale/fsl-imx8qxp-lpddr4-arm2.dts |    2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2.dts
index 2fffd99..ece42a6 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2.dts
@@ -395,7 +395,7 @@
 	phy-handle = <&ethphy1>;
 	fsl,ar8031-phy-fixup;
 	fsl,magic-packet;
-	status = "disabled";
+	status = "okay";
 };
 
 &flexspi0 {
diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qxp-lpddr4-arm2.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qxp-lpddr4-arm2.dts
index 30f5eb1..10cf0d1 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qxp-lpddr4-arm2.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qxp-lpddr4-arm2.dts
@@ -303,7 +303,7 @@
 	phy-handle = <&ethphy1>;
 	fsl,ar8031-phy-fixup;
 	fsl,magic-packet;
-	status = "disabled";
+	status = "okay";
 };
 
 &i2c0_csi0 {
-- 
1.7.9.5

